services/manage.ps1

```diff
@@ -96,12 +96,17 @@ function dev {
 function shell {
     docker run --rm -u ayonuser -ti -v "$CurrentDir/$ServiceDir/$ServiceDir" + ":/service:Z" $Image /bin/sh
 }
-function dist {
+function push {
```

Comment:
`dist` is quite standard name for distributing the image, why changed to `push`?